sql/moz-fx-data-shared-prod/contextual_services_derived/event_aggregates_check_v1/metadata.yaml

```diff
@@ -0,0 +1,26 @@
+friendly_name: Event Aggregates Check
+description: |-
+  This query checks whether there are any empty partitions in
+  contextual_services_derived.event_aggregates_check_v1 which might
+  indicate that data is missing.
+owners:
+- wstuckey@mozilla.com
+email:
+- rburwei@mozilla.com
+- ctroy@mozilla.com
+- wstuckey@mozilla.com
+- skahmann@mozilla.com
+- xluo@mozilla.com
```

Comment:
I just selected these email adresses based on the ones listed for existing tasks.